packages/main/src/Menu.ts

```diff
@@ -234,6 +238,9 @@ class Menu extends UI5Element {
 	@property({ converter: DOMReferenceConverter })
 	opener?: HTMLElement | string;
 
+	@property({ type: Object })
+	_parent?: MenuItem | Menu;
```

Comment:
Why do we need a parent property? I don't think that any component should know the parent? If it needs any information provided by the parent, it should go trough property and passed trough the template.

Reply:
After discussion we managed to find a way to remove this property and related code.